src/styles/components/_usage-guidelines.scss

```diff
@@ -0,0 +1,42 @@
+$breakpoint-tablet: 1220px;
+
+.cmp-usage-guidelines {
+  background-color: #01f5ac;
+
+  &__heading {
+    position: relative;
+    font-family: Viga, sans-serif;
+    font-weight: 400;
+    font-size: map.get($font-sizes, "md");
+  }
+
+  &__content {
+    padding: 0 0 6rem 3rem;
+  }
+}
+
+@media only screen and (min-width: $breakpoint-tablet) {
+  .cmp-usage-guidelines {
+    background-color: #01f5ac;
+    background-image: url("../../../public/usage-guidelines-image.svg");
+    background-repeat: no-repeat;
+    background-position: right;
+    background-size: 20rem;
+    margin: auto;
+    max-width: 59rem;
```

Comment:
The margin is preventing the background color from stretching across the length of the screen. It might be better to use padding instead and give it 100% width.
```suggestion
    margin: 0;
    width: 100%;
    padding-left: 4rem;
```

Reply:
Wow Bennett--check out what made it work!!

https://developer.mozilla.org/en-US/docs/Web/CSS/background-position

![image](https://user-images.githubusercontent.com/49820744/198639439-8efbc87a-b825-45d6-93d2-3e5d55324ef4.png)

Reply:
![image](https://user-images.githubusercontent.com/49820744/198639815-c667331a-137d-47fb-b481-d4bdd3504454.png)